relayer-engine/src/storage/redisStore.ts

```diff
@@ -61,6 +62,9 @@ async function createConnection(
         host: redisHost,
         port: redisPort,
       },
+      isolationPoolOptions: {
```

Comment:
don't know if this does anything, but since we use `blmove` that blocks the connection, we really need multiple connections